README.md

```diff
@@ -90,10 +101,13 @@ def mystery5(s)
 end
 ```
 
-- What is mystery5("hi")?
-- What is mystery5("")?
-- What is mystery5("Hi, there!")?
+ What is mystery5("hi")? "**" **
+- What is mystery5("")? ""
+- What is mystery5("Hi, there!")? = "**********"
 - _Added Fun: How could we make only alphabetic characters to be changed to stars?_
+This method is creating a string of stars * that replace every character in the string.
+
+You could add a statement: if s[0].include?(a..z) or [a-zA-Z] before running the return statement to make sure the space has a letter.  Otherwise you could have it return a space " " + mystery5(s[1..-1]).  Although this is using ruby magic and I'm not sure how to do it without.  I'd love to know!
 
```

Comment:
This looks like a good solution to me. Other languages have similar things that allow you to determine if something is a letter or not, so I wouldn't consider this ruby magic. I don't know how you'd know that until you've been exposed to other languages more though.

One question to think about is would you just strip out all none alphabetic characters? Or would you leave them in place? i.e. would the rest of mystery5modifed("Hi, there!") result in "*******" or "**, *****!" I was thinking the latter, but looks like you might have been thinking the former.